Makefile

```diff
@@ -170,7 +170,7 @@ olm-install:
 	operator-sdk olm install --timeout=5m
 
 olm-generate-package-manifests: ## OPERATOR OLM CSV - Generate CSV Manifests
-	operator-sdk generate csv --csv-version=$(RELEASE) --update-crds --make-manifests=false --csv-channel=alpha
+	operator-sdk generate csv --csv-version=$(RELEASE) --update-crds --make-manifests=false --csv-channel=alpha --from-version=$(FROM_RELEASE)
```

Comment:
Does this new env var `FROM_RELEASE` need to be initialized at he beginning of the Makefile?

Reply:
No, you should specify it always in the command line when generating a new set of olm manifests, to indicate the release you want to upgrade from. Anyway all of this is going away pretty soon with the upcoming change to the bundle format.